ibis/backends/duckdb/tests/test_register.py

```diff
@@ -383,8 +383,10 @@ def test_memtable_with_nullable_pyarrow_not_string(con):
 
 def test_set_temp_dir(tmp_path):
     path = tmp_path / "foo" / "bar"
-    ibis.duckdb.connect(temp_directory=path)
-    assert path.exists()
+    con = ibis.duckdb.connect(temp_directory=path, max_memory="1MB")
+    con.raw_sql("CREATE TABLE big AS SELECT range(100_000) as i")
```

Comment:
Ah, I see.

Thanks for adding this test, I agree this is much better than the previous one.